mappings/net/minecraft/util/math/noise/OctavePerlinNoiseSampler.mapping

```diff
@@ -1,5 +1,14 @@
 CLASS net/minecraft/class_3537 net/minecraft/util/math/noise/OctavePerlinNoiseSampler
 	FIELD field_15744 octaveSamplers [Lnet/minecraft/class_3756;
+	METHOD <init> (Lnet/minecraft/class_2919;Lit/unimi/dsi/fastutil/ints/IntSortedSet;)V
+		ARG 1 rand
+		ARG 2 octaves
+	METHOD <init> (Lnet/minecraft/class_2919;Ljava/util/List;)V
+		ARG 1 rand
+		ARG 2 octaves
+	METHOD <init> (Lnet/minecraft/class_2919;Ljava/util/stream/IntStream;)V
+		ARG 1 rand
+		ARG 2 octaves
```

Comment:
Hmm, should we use `random` or `rand`? Most code (at least in blocks/items) uses `random`.

Reply:
@EarthComputer though if you name it `random`, the name will appear red, you have actually successfully named it that way.

Reply:
Hmm, I prefer the name `rand`, but if there's a standard I will change it.

Reply:
Yet another enigma issue (not showing it as green for outside classes named) so you might've been confused.

Reply:
![rand](https://user-images.githubusercontent.com/6596629/73967224-29f62600-4920-11ea-8277-ee275b709da1.png)
There seems to only be one `rand` in some 1.15 branch in my repo, and 205 `random`s.